Backendless/Persistence/DataQueryBuilder.cs

```diff
@@ -60,9 +60,17 @@ public List<String> GetProperties()
       return properties;
     }
 
-    public DataQueryBuilder SetProperties( List<String> properties )
+    public DataQueryBuilder SetProperties( List<String> properties, params String[] property )

```

Comment:
this signature doesn't quite make sense. Why would a user pass a collection to us AND a then a variable number of values?